src/CarbonAware.DataSources/CarbonAware.DataSources.Json/src/CarbonAware.DataSources.Json.csproj

```diff
@@ -10,18 +10,12 @@
     <Nullable>enable</Nullable>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
   </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
-  </ItemGroup>
   
   <ItemGroup>
     <None Remove="sample-emissions-data.json" />
-    <None Remove="test-data-azure-emissions.json" />
   </ItemGroup>
 
   <ItemGroup>
     <EmbeddedResource Include="..\..\..\data\data-files\sample-emissions-data.json" />
-    <EmbeddedResource Include="..\..\..\data\data-files\test-data-azure-emissions.json" />
```

Comment:
I assume that this works, but can you help me understand why we don't need this anymore if it's the default value? 

And also why are we keeping the other file?

Reply:
It is not needed since now we are allowing the user to specify a sample file. Before we were bundling the file as part of the resource manager, which is very convenient for files that don't change (i.e resource string).
With regards `sample-emissions...`, I'll remove it from this reference, since it is not used anywhere.